### DISCUSSION TITLE
Can't commit or see commits , error

### DISCUSSION THREAD

**Discussion by kaaameeehaaameeehaaa** [community/community#55182] [Actions] [answered]
### Select Topic Area

Question

### Body

So I was pushing different files to 2 different GitHub profiles and seems like I got banned. Can that be ? Or maybe GitHub thought im bot so my IP was banned? :D

**Comment by s0m31-hub** [comment, marked as answer]
https://www.githubstatus.com/incidents/nf7s6933tnn8

**Reply by kaaameeehaaameeehaaa** [reply]
Thanks!

**Comment by Abhiram716** [comment]
I am commiting into my repo but I cant see commits in my contribution graph.can anyone tell me what's the reason

**Reply by s0m31-hub** [reply]
Are you talking about profile activity? I can see your contributions there
![image](https://github.com/community/community/assets/45433883/bcfe4823-9c8f-471c-a4aa-5a6ba8cbd864)

**Reply by Abhiram716** [reply]
On june 8 and june 7 I have made few more contributions than It's in my contribution graph, I can't figure out what's the reason.

**Comment by vvronskyFX** [comment]
Hey there,

I've recently been committing to a Repo that i've created. i didn't fork it or anything. But I've been trying to figure out why the Commits do not show up in my contribution graph on my profile. 

My emails on my GitHub and project repo are the same. It's not forked. All the files are being committed from my local file location. 

Here's my repo I've been committing to so far: https://github.com/vvronskyFX/javascript-projects-lab/tree/main/weather-app

Here's my profile with the contribution graph: https://github.com/vvronskyFX

Any advice on what could be happening and solutions?

**Reply by vvronskyFX** [reply]
Some more info - 

When I add ".patch" to the end of my recent commit URL Is get this: 
![Screenshot 2023-06-18 at 3 14 48 PM](https://github.com/community/community/assets/64465297/20649fd4-30d2-45fc-8f3f-9eca4daee221)

However, I've set my email in my Terminal like so:
<img width="1608" alt="Screenshot 2023-06-18 at 3 13 03 PM" src="https://github.com/community/community/assets/64465297/b0a25626-6021-4f17-a704-4b477fa7a481">

My GitHub primary email is set to the same:
![Screenshot 2023-06-18 at 3 12 28 PM](https://github.com/community/community/assets/64465297/a6e43abe-33f9-4175-88bf-7b9134b39845)

Am i doing something wrong here? I've tried looking into the gutHub support Docs on this and so far, no solution is found.

**Reply by airtower-luna** [reply]
Fixing the `user.email` setting won't fix already existing commits, you'll have to [rewrite repository history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to do that (with all the caveats that has). But the "email privacy" setting you showed in that screenshot would make GitHub reject any pushes with that author email address, you'll have to choose what you want. :sweat_smile: 

Also this *really* should be a separate thread, it's a completely different question than any other post in this one. :wink:
